qiskit/visualization/circuit_visualization.py

```diff
@@ -335,6 +343,11 @@ def _text_circuit_drawer(
     text_drawing.vertical_compression = vertical_compression
 
     if filename:
+        if not filename.endswith(".txt"):
+            raise ValueError(
+                "ERROR: Filename extension is not .txt. "
+                "Please use a .txt file extension to save in text format."
```

Comment:
Error messages are not multiline. I would suggest something shorter, like:
```suggestion
                "ERROR: filename parameter does not use .txt extension."
```

Reply:
Got it. Updated the file.